app/views/layouts/etm/_settings_menu.html.haml

```diff
@@ -27,11 +27,16 @@
       - if has_active_scenario?
         %a.save{:href => new_scenario_path}= t("header.save_scenario")
         %a#reset_scenario{:href => scenario_reset_path}= t("header.reset_scenario")
+
+    - if has_active_scenario?
+      %li.sep
+        = link_to t("header.show_graph"), et_engine_graph_url, target: "_blank"
```

Comment:
> Why do we want a link to ETEngine? Most people will not have an ETE account; is this for QI staff use?

This link should show people the **publicly accessible** graph of their current scenario: http://et-engine.com/data/:id/layout. So no need to log in! :smile: